index.html

```diff
@@ -156,9 +219,12 @@ <h2>Profesional Art Printing Data</h2>
           </ul>
           <button type="button" class="job-button">See Project</button>
         </div>
+      </div>
       </section>
       <section>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 3-Main (2)](https://user-images.githubusercontent.com/69638013/159862511-0d0b7e8c-4a2b-45ec-ba4a-f31ec65b850a.png)  | ![Screenshot_28](https://user-images.githubusercontent.com/69638013/159862534-2568d9b5-4069-4aec-a98d-defa4cf5b2c7.png)


- [ ] Kindly align the about me and background image to have the same width with the sublinks 👍🏿

Reply:
> Figma Design	Your design
> ![Template 3-Main (2)](https://user-images.githubusercontent.com/69638013/159862511-0d0b7e8c-4a2b-45ec-ba4a-f31ec65b850a.png)	![Screenshot_28](https://user-images.githubusercontent.com/69638013/159862534-2568d9b5-4069-4aec-a98d-defa4cf5b2c7.png)
> * [ ]  Kindly align the about me and background image to have the same width with the sublinks 👍🏿

- [ ] As requested by the previous reviewer, please make sure this paragraph is aligned as per the Figma design
<img width="1272" alt="abt" src="https://user-images.githubusercontent.com/79058364/159888836-d3ffe81c-c745-4f54-8af8-0aeaa44baf77.png">

- [ OPTIONAL] In addition to it, Please adjust your styling to match the design of your framework technologies below ⬇️ 
<img width="1172" alt="lang" src="https://user-images.githubusercontent.com/79058364/159888892-4eed3d42-6357-4850-95c6-18a9d9e89c4a.png">

- [ THIS SECTION SHOULD LOOK LIKE THIS INSTEAD]
<img width="455" alt="orgi" src="https://user-images.githubusercontent.com/79058364/159889039-075808b1-5339-4eb1-a3cd-e2f4553186a4.png">